orchagent/fdborch.cpp

```diff
@@ -168,11 +168,23 @@ void FdbOrch::update(sai_fdb_event_t        type,
                    type, update.entry.mac.to_string().c_str(),
                    entry->bv_id, bridge_port_id);
 
+
     if (bridge_port_id &&
         !m_portsOrch->getPortByBridgePortId(bridge_port_id, update.port))
     {
-        SWSS_LOG_ERROR("Failed to get port by bridge port ID 0x%" PRIx64 ".",
+        if (type == SAI_FDB_EVENT_FLUSHED)
+        {
+            /* in case of flush - can be ignored due to a race */
+            /* there are notifications about FDB FLUSH (syncd/sai_redis) on port which was already removed
+             * by orchagent as a result of removeVlanMember action (removeBridgePort) */
+            SWSS_LOG_INFO("Flush ignored. Received event on bridge port ID 0x%" PRIx64 " which is not exist.",
```

Comment:
Can you please rephrase to "Flush event: Failed to get port by bridge port ID 0x%" PRIx64 "."

Reply:
I can :) but, IMO its better to avoid to use word "Failed"in INFO notification.
What do you think?

Reply:
Agree, but this has been the model for some exceptions.. For e.g check [this](https://github.com/Azure/sonic-swss/blob/master/orchagent/fdborch.cpp#L832).  IMO, even though it is INFO, it is still a failure in fetch the data.